### DISCUSSION TITLE
Work around access biomart in run_scenicplus biomart

### DISCUSSION THREAD

**Discussion by zqzneptune** [aertslab/scenicplus#98] [General]
Hello, 

Our HPC nodes cannot access the internet. Is there any work around for `run_scenicplus` to avoid trying to get information from biomart on the fly?

Thanks!

**Comment by SeppeDeWinter** [comment]
Hi @zqzneptune 

These are the steps where biomart is used:

- get_search_space
- export_to_UCSC_interact

With the last step being optional.

In order to not use biomart you will have to run the `get_search_space` function manually prior to running the wrapper script.
To do this see: https://scenicplus.readthedocs.io/en/development/pbmc_multiome_tutorial.html#Notes-on-the-individual-function-calls-in-the-wrapper-function. `Defining the search space`.

After you have run this you should be able to continue using the wrapper function (`run_scenicplus`) without any problem (with `export_to_UCSC_file` set to `False`).

I hope this helps.

Best,

Seppe
